app/src/main/java/com/kickstarter/ui/viewholders/CreatorDashboardHeaderViewHolder.java

```diff
@@ -0,0 +1,131 @@
+  protected @Bind(R.id.creator_dashboard_funding_text) TextView fundingTextTextView;
+  protected @Bind(R.id.creator_dashboard_percent) TextView percentTextView;
+  protected @Bind(R.id.creator_dashboard_project_name) TextView projectNameTextView;
+  protected @Bind(R.id.creator_dashboard_time_remaining) TextView timeRemainingTextView;
+  protected @Bind(R.id.creator_dashboard_time_remaining_text) TextView timeRemainingTextTextView;
+  protected @BindString(R.string.discovery_baseball_card_stats_pledged_of_goal) String pledgedOfGoalString;
+
+  private KSString ksString;
+  private KSCurrency ksCurrency;
+
+  public CreatorDashboardHeaderViewHolder(final @NonNull View view) {
+    super(view);
+
+    viewModel = new CreatorDashboardHeaderHolderViewModel.ViewModel(environment());
+    ButterKnife.bind(this, view);
+
+    this.ksCurrency = this.environment().ksCurrency();
+    this.ksString = this.environment().ksString();
+
+    viewModel.outputs.currentProject()
+      .compose(bindToLifecycle())
+      .compose(observeForUI())
+      .subscribe(this::setTimeRemainingTextTextView);
+
+    viewModel.outputs.currentProject()
+      .compose(bindToLifecycle())
+      .compose(observeForUI())
+      .subscribe(this::setPledgedOfGoalString);
+
+    viewModel.outputs.currentProject()
+      .compose(bindToLifecycle())
+      .compose(observeForUI())
+      .subscribe(this::setPledgedOfGoalString);
+
+    viewModel.outputs.percentageFunded()
+      .compose(bindToLifecycle())
+      .compose(observeForUI())
+      .subscribe(percentTextView::setText);
+
+    viewModel.outputs.projectBackersCountText()
+      .compose(bindToLifecycle())
+      .compose(observeForUI())
+      .subscribe(backerCountTextView::setText);
+
+    viewModel.outputs.projectNameTextViewText()
+      .compose(bindToLifecycle())
+      .compose(observeForUI())
+      .subscribe(projectNameTextView::setText);
+
+    viewModel.outputs.timeRemainingText()
+      .compose(bindToLifecycle())
+      .compose(observeForUI())
+      .subscribe(timeRemainingTextView::setText);
+
+    viewModel.outputs.startProjectActivity()
+      .compose(bindToLifecycle())
+      .compose(observeForUI())
+      .subscribe(projectAndRefTag -> this.startProjectActivity(projectAndRefTag.first, projectAndRefTag.second));
+  }
+
+  @OnClick(R.id.creator_view_project_button)
+  protected void viewProjectButtonClicked() {
+    this.viewModel.inputs.projectViewClicked();
+  }
+
+  @Override
+  public void bindData(final @Nullable Object data) throws Exception {
+    // coerce to projectstats and project
+    final Pair<Project, ProjectStats> projectAndProjectStats = requireNonNull((Pair<Project, ProjectStats>) data);
+    final Project project = requireNonNull(projectAndProjectStats.first, Project.class);
+    final ProjectStats projectStats = requireNonNull(projectAndProjectStats.second, ProjectStats.class);
+    viewModel.inputs.projectAndStats(project, projectStats);
```

Comment:
since we're using these values as a pair anyways in the VM, we can just go ahead and make the `projectAndStats` input a `Pair<Project, ProjectStats>` type so we're not destructuring / restructuring the pair!